build.sbt

```diff
@@ -375,14 +375,14 @@ def sharedNativeImageSettings(targetDir: File) = Seq(
     "akka.dispatch.affinity",
     "akka.util",
     "com.google.Protobuf",
+    "com.typesafe.config",
     "java.lang.ref.SoftReference", // https://github.com/oracle/graal/issues/2345
     "java.lang.invoke.MethodHandleImpl" // https://github.com/oracle/graal/issues/2345
   ).mkString("=", ",", ""),
-  "--initialize-at-run-time=" +
+  "-H:ClassInitialization=com.typesafe.config.impl.ConfigImpl$EnvVariablesHolder:rerun",
```

Comment:
We can revert these to runtime once/if we decide to drop the serializer-automaticfeature. They are there to make sure that the system properties, environment variables, etc are loaded at runtime and not at build time.